DESCRIPTION

```diff
@@ -10,7 +10,19 @@ Description: Implements an individual-based model of COVID-19 transmission.
 License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
+Remotes:  
+    mrc-ide/individual@0.0.8,
+    mrc-ide/squire@v0.5.3
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 Suggests:
-    testthat
+    testthat,
+    mockery
+Imports:
```

Comment:
Can you strip this down to the imports that are actually used? I believe that is only `individual` at the moment, with `squire` in Suggests as it's used in the tests